CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [unreleased]
+### Security
+### Added
+### Changed
+### Removed
+
+## [v0.7.0] - 26/11/2021
+### Security
+- dependencies update ([#6](https://github.com/flatcar-linux/locksmith/pull/6))
+- etcd, dbus and systemd Go dependencies update ([#8](https://github.com/flatcar-linux/locksmith/pull/8))
+
+### Added
+
+### Changed
+- ignore nil sent by dbus.Signal in the daemon ([#1](https://github.com/flatcar-linux/locksmith/pull/1))
```

Comment:
This is really a fix description, not what it fixes. It actually fixes a possible panic on daemon shutdown.